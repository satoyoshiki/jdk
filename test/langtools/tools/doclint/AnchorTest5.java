/*
 * @test /nodynamiccopyright/
 * @bug 8256312
 * @summary Valid anchor 'id' value not allowed
 * @modules jdk.javadoc/jdk.javadoc.internal.doclint
 * @build DocLintTester
 * @run main DocLintTester -XhtmlVersion:html5 -ref AnchorTest5.out AnchorTest5.java
 */

/** */
public class AnchorTest5 {
    /**
     * <a id=123 ></a>
     */
    public void a_id_valid() { }

    /**
     * <a id="123 "></a>
     */
    public void a_id_invalid() { }

    /**
     * <a id=&#x2615></a>
     */
    public void a_id_valid_characters() { }

    /**
     * <p id=123 >text</p>
     */
    public void p_id_already_defined() { }

}
