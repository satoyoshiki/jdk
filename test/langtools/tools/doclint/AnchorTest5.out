AnchorTest5.java:18: error: invalid name for anchor: "123 "
     * <a id="123 "></a>
          ^
AnchorTest5.java:28: error: anchor already defined: "123"
     * <p id=123 >text</p>
          ^
2 errors
